lib/pull-request/merge-back.ts

```diff
@@ -81,7 +87,9 @@ export interface AutoMergeBackOptions extends pr.AutoPullRequestOptions {
    * @default - no condition
    */
   condition?: string;
+}
 
+export interface AutoMergeBackOptions extends pr.AutoPullRequestOptions, CommonMergeOptions {
```

Comment:
I believe multiple inheritance is not jsii supported. While delivlib is not jsii enabled, might be nice to try to strive for that modeling.

Reply:
On second look - it looks like `CommonMergeOptions` already extends `AutoPullRequestOptions` so probably just drop the redundant one.

Reply:
Yeah, I'll drop the redundant `AutoPullRequestOptions` inheritence; that was a mistake.